File: SGA_Api/SGA_Api/Models/Traspasos/FinalizarTraspasoPaletDto.cs
namespace SGA_Api.Models.Traspasos
{
    public class FinalizarTraspasoPaletDto
    {
        public string AlmacenDestino { get; set; }
        public string UbicacionDestino { get; set; }
        public int UsuarioFinalizacionId { get; set; }
        public string CodigoEstado { get; set; }
    }
} 

File: SGA_Android/app/src/main/java/com/example/sga/data/dto/traspasos/FinalizarTraspasoPaletDto.kt
package com.example.sga.data.dto.traspasos

data class FinalizarTraspasoPaletDto(
    val almacenDestino: String,
    val ubicacionDestino: String,
    val usuarioFinalizacionId: Int,
    val codigoEstado: String = "PENDIENTE_ERP"
)
